lighthouse-core/report/v2/renderer/report-ui-features.js

```diff
@@ -71,6 +73,25 @@ class ReportUIFeatures {
     this._document.dispatchEvent(event);
   }
 
+  _setupMediaQueryListeners() {
+    const mediaQuery = self.matchMedia('(max-width: 600px)');
+    mediaQuery.addListener(this.onMediaQueryChange);
+    this.onMediaQueryChange(mediaQuery);
```

Comment:
are you calling this to kick it on init or is this for testing? If it's the former, can you add a comment?